### PR TITLE
#720  - Remove 'www.' from domain request form

### LINKED ISSUE #720: Remove “www” from domain input field on domain request form
For consistency throughout the request experience, please remove "www." from this page, as it does not appear in other contexts.

Related to #763

### PR DESCRIPTION
**Pull request by zandercymatics**
## Ticket

Resolves #720 

## Changes
- Removed 'www.' from the input boxes on domain request form
- Changed the variable 'www_gov' into  'append_gov"
- removed www_gov=True from alternate_domain and domain and replaced it with append_gov=True

## Context for reviewers
- Instead of commenting part out the 'www_gov' implementation, I've decided to split it up into two variables instead, so we can easily add this back if we change our mind. It also gives more flexibility if we need it.

## Setup
Create a new domain request as normal, and navigate to '.gov domain'

#### Satisfied acceptance criteria and met development standards

- [x] Met the acceptance criteria, or will meet them in a subsequent PR
- [ ] Created/modified automated tests
- [x] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [x] Messaged on Slack or in standup to notify the team that a PR is ready for review
- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Original Developer)

- [ ] All new functions and methods are commented using plain language
- [ ] Did dependency updates in Pipfile also get changed in requirements.txt?
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values

#### Validated user-facing changes (if applicable)

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [x] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] Add at least 1 designer as PR reviewer

#### Reviewed, tested, and left feedback about the changes

- [x] Pulled this branch locally and tested it
- [x] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Code reviewer)

- [ ] All new functions and methods are commented using plain language
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values
- [ ] (Rarely needed) Did dependency updates in Pipfile also get changed in requirements.txt?

#### Validated user-facing changes as a developer

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Meets all designs and user flows provided by design/product
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers, the suggestion is to use ones that the developer didn't (check off which ones were used)
  - [x] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

#### Verified that the changes match the design intention

- [ ] Checked that the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow

#### Validated user-facing changes as a designer

- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

## Screenshots

image

### CODE REVIEW

File: src/registrar/templates/includes/input_with_errors.html [hunk 1]
```diff
@@ -55,16 +55,20 @@
     </div>
   {% endif %}
 
-  {% if www_gov %}
+  {% if prepend_www or append_gov %}
```
Comment (rachidatecs) [inline src/registrar/templates/includes/input_with_errors.html:58]: This seems redundant

File: src/registrar/templates/includes/input_with_errors.html [hunk 2]
```diff
@@ -55,16 +55,20 @@
...
     <div class="display-flex flex-align-center">
+      {# ticket #720: https://github.com/cisagov/getgov/issues/720 #}
```
Comment (rachidatecs) [inline src/registrar/templates/includes/input_with_errors.html:60]: Remove this comment

File: src/registrar/templates/includes/input_with_errors.html (hunk 1 above)
Comment (rachidatecs) [reply src/registrar/templates/includes/input_with_errors.html:58]: Disregard, I see it (the wrapping tag)

**Comment by rachidatecs** [comment]
Since we're not using or planning on using www., you can simplify the template 

```
{% if append_gov %}
    <div class="display-flex flex-align-center">
  {% endif %}
  {# this is the input field, itself #}
  {% include widget.template_name %}
  {% if append_gov %}
      <span class="padding-top-05 padding-left-2px">.gov </span>
    </div>
  {% endif %}
```

**Comment by zandercymatics** [comment]
@rachidatecs Would it be better to just comment it out directly? In my first commit I just kept the www_gov variable and commented that portion out, so I can revert to that

**Comment by rachidatecs** [comment]
Chances are that it's gone for good, so I would just chuck it. If I'm mistaken, it's easy enough to bring back.

Your call though, none of my comments are blocking.

**Review by rachidatecs — APPROVED** [review]

**Comment by zandercymatics** [comment]
@rachidatecs Probably best for cleanliness then to remove it like you said. Thanks for the feedback on this

**Review by therealslimhsiehdy — APPROVED** [review]

**Pull request merged**